spark/v3.2/spark/src/main/java/org/apache/iceberg/spark/actions/DeleteOrphanFilesSparkAction.java

```diff
@@ -326,6 +346,96 @@ private static FlatMapFunction<Iterator<String>, String> listDirsRecursively(
     };
   }
 
+  @VisibleForTesting
+  static List<String> findOrphanFiles(
+          SparkSession spark, Dataset<Row> actualFileDF, Dataset<Row> validFileDF,
```

Comment:
nit: this also uses 8 spaces instead of 4

Reply:
fixed